GMG/GMG/Core/Models/MusicNotation/TimeSignature.swift

```diff
@@ -1,6 +1,6 @@
 //  Copyright © 2025 ADA 4th GMG. All rights reserved.
 
-struct TimeSignature {
+struct TimeSignature: Equatable {
```

Comment:
extension으로 빼는 것 어떨까요?

```suggestion
extension TimeSignature: Equatable {}
```

Reply:
수정했습니다!